devices/models.py

```diff
@@ -100,6 +102,35 @@ class Bookmark(models.Model):
     user = models.ForeignKey(Lageruser)
 
 
+class Vendor(models.Model):
+    name = models.CharField(_('Name'), max_length=100)
+    driver_url = models.CharField(_('Drivers'), max_length=1000, blank=True, null=True)
```

Comment:
why not `model.URLField` ?

Reply:
also, what's the difference between Vendor and Manufacturer?

Reply:
I didn't notice that there was already a manufacturer :S ok, changing vendor -> manufacturer